Source/WSTagsField.swift

```diff
@@ -169,6 +187,11 @@ open class WSTagsField: UIView {
         }
     }
     
+    public var minNumberOfCharacters: Int?
+    public var maxNumberOfCharacters: Int?
+    public var minCharacterValidationMessage: String = "You did not enter enough characters for a valid tag."
```

Comment:
isn't there a better way to identify them ... enum .. a struct with constants that you can extend? some protocol ...